src/components/GameItem.js

```diff
@@ -4,23 +4,43 @@ import ItemIcons from '../ItemIcons.js';
 import PropTypes from 'prop-types';
 
 class GameItem extends Component {
-  propTypes = {
+  static propTypes = {
     height: PropTypes.number.isRequired,
     layer: PropTypes.number.isRequired,
   }
 
+  constructor(props) {
+    super(props);
+    this.state = {
+      clicked: false
+    };
+  }
+
+  clickedItem = () => {
+    this.setState({
+      clicked: true,
+    });
+    this.props.type === 'litter' && this.props.clickedItemClickCallback(this.props.index);
+  }
+
   render() {
     const itemStyle = {
       bottom: `${this.props.height}px`, // use props.height to offset from the bottom of screen
       zIndex: this.props.layer, // use props.layer to set z-index, so we display ontop of background
     };
 
     // Update this to select the correct icon for each item
-    const icon = ItemIcons.rock;
+    let clickedItemStyle = '';
+
+    if (this.state.clicked) {
+      this.props.type === 'litter' ? clickedItemStyle = ' spotted-litter' : clickedItemStyle = ' spotted-nature';
+    }
+    
+    const icon = ItemIcons[this.props.type];
 
     return (
-      <div className="game-item" style={itemStyle}>
-        <img src={icon} alt="Item" className="icon-item"></img>
+      <div className={`game-item${clickedItemStyle}`} style={itemStyle} onClick={this.clickedItem}>
```

Comment:
Instead of relying on `clickedItemStyle` to have a space, it may make sense to put the space in this string, so it's just interpolated in:
```
className={`game-item ${clickedItemStyle}`}
```